daemon/src/oracle.rs

```diff
@@ -36,12 +36,14 @@ pub struct Actor {
     announcements: HashMap<BitMexPriceEventId, (OffsetDateTime, Vec<schnorrsig::PublicKey>)>,
     pending_attestations: HashSet<BitMexPriceEventId>,
     executor: command::Executor,
-    announcement_lookahead: Duration,
     tasks: Tasks,
     db: sqlite_db::Connection,
     client: reqwest::Client,
 }
 
+/// We want to fetch at least this much announcements into the future
+const ANNOUNCEMENT_LOOKAHEAD: Duration = Duration::hours(26);
```

Comment:
🙃 I liked it better when we saw the dependency on `SETTLEMENT_INTERVAL` directly here, i.e.:

```
const ANNOUNCEMENT_LOOKAHEAD: Duration = SETTLEMENT_INTERVAL + Duration::hours(2);
```

The test ensures that it is, but when just looking at `ANNOUNCEMENT_LOOKAHEAD` it is not clear why it is set to `26`.

Reply:
+1 - it was more clear, especially if we ever decide to change `SETTLEMENT_INTERVAL`.

Reply:
I had that at the beginning but rust does not allow it.